docs/_source/_common/tabs/argilla_install.md

```diff
@@ -15,6 +15,9 @@ wget -O docker-compose.yaml https://raw.githubusercontent.com/argilla-io/argilla
 :::
 
 :::{tab-item} Hugging Face Spaces
+```{warning}
+HuggingFace Spaces don't have persistent storage and will automatically reset after 48 hours of inactivity, which means all the data within the Space will be lost. You can enable this via the Hugging Face Spaces settings.
```

Comment:
```suggestion
HuggingFace Spaces now have persistent storage and this is supported from Argilla 1.11.0 onwards, but you will need to manually activate it via the HuggingFace Spaces settings. Otherwise, unless you're on a paid space upgrade, after 48 hours of inactivity the space will be shut off and you will lose all the data. To avoid losing data, we highly recommend using the persistent storage layer offered by HuggingFace.
```